docs/android/linter.md

```diff
@@ -139,7 +139,13 @@ After updating, review the ignored errors to determine if they should be address
 
 ## Extending lint rules
 
-We encourage you to propose new linter rules specific to our project. These rules can help identify misuse of APIs or enforce design patterns.
+We encourage you to propose new linter rules specific to our project. These rules can help identify misuse of APIs or enforce design patterns that we want to be enforced in the project.
```

Comment:
Just a rewrite to avoid repeating the word "enforce" or a variation of it.
```suggestion
We encourage you to propose new linter rules specific to our project. These rules can help identify misuse of APIs or enforce design patterns that we want used in the project.
```